libcaf_net/caf/net/tcp_accept_socket.hpp

```diff
@@ -0,0 +1,75 @@
+/******************************************************************************
+ *                       ____    _    _____                                   *
+ *                      / ___|  / \  |  ___|    C++                           *
+ *                     | |     / _ \ | |_       Actor                         *
+ *                     | |___ / ___ \|  _|      Framework                     *
+ *                      \____/_/   \_|_|                                      *
+ *                                                                            *
+ * Copyright 2011-2019 Dominik Charousset                                     *
+ *                                                                            *
+ * Distributed under the terms and conditions of the BSD 3-Clause License or  *
+ * (at your option) under the terms and conditions of the Boost Software      *
+ * License 1.0. See accompanying files LICENSE and LICENSE_ALTERNATIVE.       *
+ *                                                                            *
+ * If you did not receive a copy of the license files, see                    *
+ * http://opensource.org/licenses/BSD-3-Clause and                            *
+ * http://www.boost.org/LICENSE_1_0.txt.                                      *
+ ******************************************************************************/
+
+#pragma once
+
+#include "caf/net/abstract_socket.hpp"
+#include "caf/net/network_socket.hpp"
+#include "caf/net/socket.hpp"
+#include "caf/net/stream_socket.hpp"
+#include "caf/net/tcp_stream_socket.hpp"
+#include "caf/uri.hpp"
+
+namespace caf {
+namespace net {
+
+/// Represents an open TCP endpoint. Can be implicitly converted to a
```

Comment:
I would drop the remarks on implicit conversions. Also, I find "open TCP endpoint" a bit confusion. Why not simply this?

```suggestion
/// Represents a TCP acceptor in listening mode.
```